app/models/manageiq/providers/amazon/cloud_manager/metrics_capture.rb

```diff
@@ -1,66 +1,96 @@
+# frozen_string_literal: true
+
 class ManageIQ::Providers::Amazon::CloudManager::MetricsCapture < ManageIQ::Providers::BaseManager::MetricsCapture
-  INTERVALS = [5.minutes, 1.minute]
+  INTERVALS = [5.minutes.freeze, 1.minute.freeze].freeze
```

Comment:
@AlexanderZagaynov btw, it's usually a good idea to have the rubocop fixes in another commit, while starting with a consistent style, just adding the feature. 

This PR will probably get backported, so then it can be less pain. In some cases I wouldn't even backport a commit with style changes, since there is possibility of more conflicts, since the style changes are across many lines.